app/wiring/Assets.scala

```diff
@@ -4,5 +4,5 @@ import assets.AssetsResolver
 import play.api.BuiltInComponentsFromContext
 
 trait Assets { self: BuiltInComponentsFromContext =>
-  val assetsResolver = new AssetsResolver("/assets/", "assets.map", environment)
+  val assetsResolver = new AssetsResolver("", "assets.map", environment)
```

Comment:
The newest version of webpack-manifest-plugin includes  `/assets/` in the output so we need to remove this base path or it gets doubled up